src/main/java/com/thullo/service/TaskServiceImpl.java

```diff
@@ -71,14 +71,14 @@ private String uploadTaskFile(MultipartFile file, String requestUrl) throws BadR
 
     @Override
     public Task moveTask(String boardRef, String status, Long position) throws ResourceNotFoundException {
-        Status statusToUpdateTo = Status.getStatus(status);
```

Comment:
In some cases, the status we receive from a request might be in lowercase, such as `in progress`. To ensure consistency, we need to convert it to uppercase, such as `IN_PROGRESS`. Could you please create a helper function that can perform this conversion? I mentioned something related to this in a previous comment.

Reply:
> Noted, So the helper function would ensure the consistency of the naming; I have some few example cases below for your review;
> 
> in progress = IN_PROGRESS inprogress = INPROGRESS (I can't split the word here, So I just return the Uppercase as a word) IN PROGRESS = IN_PROGRESS

`INPROGRESS` or `inprogress` should always result to `INPROGRESS`

Reply:
> Hello Chief,
> 
> Ok, Just what I had in mind... I'll proceed with the implementation. Please how are you able to write those examples with a border-radius?
> 
> Thanks for the update.

Encapsulate the word in this symbol ` `hello``